types/zarinpal-checkout/index.d.ts

```diff
@@ -0,0 +1,61 @@
+// Type definitions for zarinpal-checkout 0.2
+// Project: https://github.com/siamak/zarinpal-checkout
+// Definitions by: Omid Seyfan <https://github.com/iamomiid>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+declare namespace ZarinPal {
+    interface Authroity {
+        Authority: string;
+        Amount: string;
+        Channel: string;
+        Date: string;
+    }
+
+    interface PaymentRequestInput {
+        Amount: number;
+        CallbackURL: string;
+        Description: string;
+        Email?: string;
+        Mobile?: string;
+    }
+
+    interface PaymentRequestOutput {
+        status: number;
+        authority: string;
+        url: string;
+    }
+
+    interface PaymentVerificationInput {
+        Amount: number;
+        Authority: string;
+    }
+
+    interface PaymentVerificationOutput {
+        status: number;
+        RefID: number;
+    }
+
+    interface UnverifiedTransactionsOutput {
+        status: number;
+        authorities: Authroity[];
+    }
+
+    interface RefreshAuthorityInput {
+        Expire: number;
+        Authority: string;
+    }
+
+    interface RefreshAuthorityOutput {
+        status: number;
+    }
+
+    interface ZarinPalInstance {
+        PaymentRequest(input: PaymentRequestInput): Promise<PaymentRequestOutput>;
+        PaymentVerification(input: PaymentVerificationInput): Promise<PaymentVerificationOutput>;
+        UnverifiedTransactions(): Promise<UnverifiedTransactionsOutput>;
+        RefreshAuthority(input: RefreshAuthorityInput): Promise<RefreshAuthorityOutput>;
+    }
+}
+
+declare const zarinpal: { create(merchantID: string, sandbox: boolean): ZarinPal.ZarinPalInstance };
+
+export default zarinpal;
```

Comment:
zarinpal-checkout doesn't export a `default` property
```suggestion
export = zarinpal;
```

and should be imported with `import Z = require('zarinpal-checkout');`